onnx/defs/generator/defs.cc

```diff
@@ -896,4 +896,76 @@ ONNX_OPERATOR_SET_SCHEMA(
           }
         }));
 
+static const char* Bernoulli_ver15_doc = R"DOC(
+    Draws binary random numbers (0 or 1) from a Bernoulli distribution. The input tensor should be
+    a tensor containing probabilities to be used for drawing the binary random number.
```

Comment:
Please clarify if the input-tensor value specifies the probability of a 1 (I assume) or 0.

Reply:
This means the output of Bernoulli is 0 or 1, input tensor is a values from range [0, 1].
As [PyTorch docs description](https://pytorch.org/docs/stable/generated/torch.bernoulli.html).
I completed the description in latest commit.

Reply:
Sorry I was unclear. I suggest changing the second sentence to
```
The input tensor should be a tensor containing probabilities p (a value in the range [0,1]) to be
used for drawing the binary random number, where an output of 1 is produced with probability
p and an output of 0 is produced with probability (1-p).
```

Reply:
Done, this description is more reasonable and clearer.